### PR TITLE
react-rendering-lists

### PR DESCRIPTION
**Pull request by dannyandreev**
image

### CODE REVIEW

File: react-rendering-lists/src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+class Pokedex extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+
+
+    const listItems = pokedex.map((card) => {
+      return (<li>{card.name}</li>)
```
Comment (scbowler) [inline react-rendering-lists/src/index.jsx:21]: Your `li` needs to have a `key` prop. You can use `card.number` to set it

File: react-rendering-lists/src/index.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  { number: '004', name: 'Charmander' },
+  { number: '007', name: 'Squirtle' },
+  { number: '025', name: 'Pikachu' },
+  { number: '039', name: 'Jigglypuff' }
+];
+
+class Pokedex extends React.Component {
+  constructor(props) {
+    super(props);
+  }
```
Comment (scbowler) [inline react-rendering-lists/src/index.jsx:15]: Since you aren't doing anything with the `constructor` you don't need it at all here

**Review by scbowler — CHANGES_REQUESTED** [review]
A couple changes needed. Let us know if you have any questions

File: react-rendering-lists/src/index.jsx (hunk 2 above)
Comment (scbowler) [reply react-rendering-lists/src/index.jsx:15]: Delete the `constructor`, it is not needed in this case

File: react-rendering-lists/src/index.jsx (hunk 1 above)
Comment (scbowler) [reply react-rendering-lists/src/index.jsx:21]: Add a key prop to the `<li>`:

```JSX
return (<li key={card.number}>{card.name}</li>);
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Couple changes still needed. Let us know if you have any questions

**Comment by dannyandreev** [comment]
image

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Pull request merged**